third_party/3/jwt.pyi

```diff
@@ -0,0 +1,65 @@
+from typing import Mapping, Any, Optional, Union
+
+
+def decode(
+    jwt: Union[str, bytes],
+    key: Union[str, bytes]=...,
+    verify: bool=...,
+    algorithms: Optional[Any]=...,
+    options: Optional[Mapping[Any, Any]]=...,
+    **kwargs: Any) -> Mapping[str, Any]:
+    ...
+
+
+def encode(
+    payload: Mapping[str, Any],
+    key: Union[str, bytes],
+    algorithm: str=...,
+    headers: Optional[Mapping[str, Any]]=...,
+    json_encoder: Optional[Any]=...) -> bytes:
+    ...
+
+
+class InvalidTokenError(Exception):
```

Comment:
typeshed's formatting convention is to use more compact code: don't put `pass` on a new line, don't put empty lines between classes.

However, we want spaces around `=` for default values.